Tasks/AzurePowerShellV4/AzurePowerShell.ps1

```diff
@@ -263,5 +263,9 @@ finally {
     Import-Module $PSScriptRoot\ps_modules\VstsAzureHelpers_
     Remove-EndpointSecrets
     Disconnect-AzureAndClearContext -restrictContext $restrictContext -ErrorAction SilentlyContinue
+
+    # Telemetry
+    $telemetryJsonContent = @{ targetAzurePs = $targetAzurePs } | ConvertTo-Json
```

Comment:
Should we apply the same for azurepowershell.ts as well?

Reply:
Done